### ISSUE TITLE
The file

### ISSUE THREAD

**Issue by silverstone000** [Akshay94/wanproject#2]
Please add that file yourself, I'm not collaborator of this repository.

And put some examples of sent and received message of your program for me to check if they are compatible.

**Comment by Akshay94** [comment]
Hey man, I cannot see your repo anymore. Why is that?

**Comment by Akshay94** [comment]
I meant i am not a collab anynmore

**Comment by silverstone000** [comment]
Yes, since you don't need to update files on my repository, I remove you from collaborators. You can leave issue on my repository or update file in yours still.

**Comment by silverstone000** [comment]
And please put some examples of message sent from running program.

**Comment by Akshay94** [comment]
hey man I can t see any issues that you said you would put up.

**Comment by silverstone000** [comment]
> 
> 
> And please put some examples of message sent from running program.

You see, I asked for that information to know about the behavior of your program. For you data format, I did as what I understand to modify mine. What remains unknown is how your file transfer program register with and receive from router program, yet you didn't respond to this. And I am not  sure about if your program will send correct message to properly control router, currently I put some function to read configuration from file in case our programs can not cooperate.

 You never asked, but file about router id and socket information is stored in a directory named 'rtconfig' under the parent directory of router program's directory. Each router has a file named rx where x is the router id with content of id, ip and port separated by space.
